validate.js

```diff
@@ -27,6 +28,8 @@ export function validateInput(inputElement) {
         }
 
         handlers[inputElementType](inputElement)
+    } else if (inputElementType == "radio") {
```

Comment:
placeholder for future validation?

Reply:
I just branched this so it stopped falling into the else block and outputting 'no handle for type radio'. I think that log is useful in case issues arise with other types of elements (especially during this rewrite), but we don't need a radio handler, so I wanted to silence that one.